lang/main-fr.json

```diff
@@ -65,26 +65,26 @@
         "dontShowAgain": ""
     },
     "connectingOverlay": {
-        "joiningRoom": "Connexion à la réunion..."
+        "joiningRoom": "Connexion à la réunion…"
```

Comment:
Why is this change? This may break stuff ...!

Reply:
I didn't know it could break stuff. I use this translation on my instance and I don't notice any particular problem. 
I often read that it is recommended to use a single character for the ellipsis rather than 3 single points, so that the space between each is good.